.github/workflows/nightly-checks.yml

```diff
@@ -5,21 +5,56 @@ on:
     - cron: '0 7 * * *'
 
 jobs:
-  build:
-    name: Build packages and upload artifacts
-    uses: ./.github/workflows/base-build.yml
-
-  integration-nextjs:
-    name: Integration (NextJS canary)
-    needs: build
-    uses: ./.github/workflows/base-e2e.yml
-    with:
-      SCRIPT: 'E2E_APP_CLERK_JS_DIR=$RUNNER_TEMP E2E_NEXTJS_VERSION=canary npm run test:integration:nextjs'
-    secrets: inherit
+  integration-tests:
+    name: Integration Tests
+    runs-on: ${{ vars.RUNNER_NORMAL }}
+    timeout-minutes: ${{ fromJSON(vars.TIMEOUT_MINUTES_NORMAL) }}
+
+    strategy:
+      matrix:
+        test-name: [ 'nextjs' ]
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          show-progress: false
+
+      - name: Setup
+        id: config
+        uses: ./.github/actions/init
+        with:
+          turbo-signature: ${{ secrets.TURBO_REMOTE_CACHE_SIGNATURE_KEY }}
+          turbo-team: ${{ vars.TURBO_TEAM }}
+          turbo-token: ${{ secrets.TURBO_TOKEN }}
+          playwright-enabled: true
+
+      - name: Verdaccio
+        uses: ./.github/actions/verdaccio
+        with:
+          publish-cmd: |
+            if [ "$(npm config get registry)" = "https://registry.npmjs.org/" ]; then echo 'Error: Using default registry' && exit 1; else npx turbo build $TURBO_ARGS && npx changeset publish --no-git-tag; fi
+
+      - name: Install @clerk/backend in /integration
+        working-directory: ./integration
+        run: npm init -y && npm install @clerk/backend
```

Comment:
❓ Why do we need to do this?

Reply:
This is code which was simply moved from another location. I believe was present as the integration dir needs it for the test utilities vs only in the tested projects themselves. @nikosdouvlis would have more context here.

Reply:
@BRKalow /integration uses `backend` as it needs access to a `clerkClient` instance (needs to hit BAPI)
We can move the dependency into a package.json file within /integration in a next iteration